src/pages/CartPage.tsx

```diff
@@ -0,0 +1,226 @@
+import React, { useContext } from 'react';
+import CartContext from '../context/Cart.context';
+import {
+    Alert,
+    Button,
+    Grid,
+    Box,
+    Card,
+    CardMedia,
+    Container,
+    CardContent,
+    Typography,
+    ListItem,
+    ListItemText,
+    Stack,
+    FormControl,
+    InputLabel,
+    Select,
+    MenuItem,
+    OutlinedInput,
+} from '@mui/material';
+import { Link } from 'react-router-dom';
+import { ROUTES } from '../utils/static';
+
+const ITEM_HEIGHT = 48;
+const ITEM_PADDING_TOP = 8;
+const MenuProps = {
+    PaperProps: {
+        style: {
+            maxHeight: ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
+            width: 250,
+        },
+    },
+};
+
+const availableQuantity = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10];
```

Comment:
why is quantity array of max 10?
Quantity should be input type number where user can specify any number of quantity if product is available